src/extractors/jsxtag-gettext.js

```diff
@@ -0,0 +1,57 @@
+import * as t from 'babel-types';
+import gettext from './tag-gettext';
+import { PO_PRIMITIVES } from '../defaults';
+import { template2Msgid, msgid2Orig, hasExpressions,
+    ast2Str, getQuasiStr, strToQuasi, dedentStr } from '../utils';
+import { NoTranslationError } from '../errors';
+
+const { MSGSTR } = PO_PRIMITIVES;
+const NAME = 'jsxtag-gettext';
+
+function match(node, context) {
+    return t.isTaggedTemplateExpression(node) && node.tag.name === context.getAliasFor(NAME);
+}
+
+function resolveDefault(nodePath, poData, context) {
```

Comment:
Recently I have refactored resove/extract api. This code is already in master. So `resolveDefault` and `resolve` will accept `node` as a first argument. And no validation for 'Translation not found' is needed in resolve.